brands/shop/tyres.json

```diff
@@ -58,6 +58,17 @@
       "shop": "tyres"
     }
   },
+  "shop/tyres|Tire Discounters": {
+    "countryCodes": ["us"],
+    "matchTags": ["shop/tyres"],
```

Comment:
fyi I'm going to remove this `matchTags`, since it just matches itself

Reply:
Oops, I meant to move this to `shop=car_repair`.

Reply:
cool.. I ended up just putting both tags in the same matchGroup (so any of these tags will be recognized)

https://github.com/osmlab/name-suggestion-index/blob/18406f5f3070795ee6f3ea8856677d6f900037e1/config/match_groups.json#L69-L74